lms/templates/instructor/instructor_dashboard_2/metrics.html

```diff
@@ -146,6 +146,30 @@
         metrics_overlay.find('.metrics-student-grades').show();
         metrics_overlay.show();
       });
+      
+      // Click handler for x axis major ticks
+      $('.metrics-container').on("click", '.stacked-bar-graph-axis .tick.major', function () {
+
+        var stackedBarElement;
+        var xValue;
+        var moduleId;
+        var tickIdentifier = d3.select(this).text();
+
+        var stackedBars = $(this).parent().siblings();
+        for (var sbIndex in stackedBars) {
```

Comment:
The `for .. in` construct in JavaScript has caveats and doesn't work like everyone expects;
it's not as straight-forward as Python.

It's probably best to just use the standard `for(i=0; i<length; i++){}` form here.

Additional arguments can be found here: http://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-a-bad-idea